src/rpc/blockchain.cpp

```diff
@@ -2099,13 +2099,16 @@ static RPCHelpMan scantxoutset()
     const std::string EXAMPLE_DESCRIPTOR_RAW = "raw(76a91411b366edfc0a8b66feebae5c2e25a7b6a5d1cf3188ac)#fm24fxxy";
 
     return RPCHelpMan{"scantxoutset",
-        "\nScans the unspent transaction output set for entries that match certain output descriptors.\n"
+        "\nScans the unspent transaction output set for entries that match certain output descriptors or miniscripts.\n"
```

Comment:
nit: i don't think it's worth mentioning Miniscript here, and especially not as something external to descriptors.

Reply:
hmm true, will remove

Reply:
@instagibbs Are you planning on changing this?

Reply:
yeah, pushed the change.